recipe/meta.yaml

```diff
@@ -24,11 +25,23 @@ requirements:
 test:
   imports:
     - tzlocal
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
+  description: |
+    This Python module returns a tzinfo object (with a pytz_deprecation_shim, 
+    for pytz compatibility) with the local timezone information, 
+    under Unix and Windows.
+  doc_url: https://github.com/regebro/tzlocal/blob/{{ version }}/CHANGES.txt
+  doc_source_url: https://github.com/regebro/tzlocal/tree/{{ version }}
```

Comment:
we have doc_url, no need for doc_source_url.  In general "doc_source_url" is pretty much superflous if there is already a doc_url tag

Reply:
This has been removed